boxes/CMakeLists.txt

```diff
@@ -1,3 +1,30 @@
+ADD_CUSTOM_COMMAND(
+    OUTPUT
+        minimal.tar.gz busybox.tar.gz python2.tar.gz python3.tar.gz
+    COMMAND
+        ${CMAKE_SOURCE_DIR}/boxes/make_boxes.sh
+    DEPENDS
+        make_boxes.sh
+    )
+
+ADD_CUSTOM_COMMAND(
+    OUTPUT
+        boxes_extraction_stamp.txt
+    COMMAND
+        tar -xvf minimal.tar.gz
+            && tar -xvf busybox.tar.gz
+            && tar -xvf python2.tar.gz
+            && tar -xvf python3.tar.gz
+            && touch boxes_extraction_stamp.txt
+    DEPENDS
+        minimal.tar.gz busybox.tar.gz python2.tar.gz python3.tar.gz
+    )
```

Comment:
Dobrze rozumiem, że on je najpierw generuje w katalogach, potem taruje, a potem roztarowuje spowrotem w to samo miejsce?

Reply:
Nie wiem czy to samo miejsce, ale z mniej-więcej tak ;p. Za to pozwala na zamianę budowania na pobieranie, albo na inne budowanie (czy tam na podanie jakiś opcji do tara).

Reply:
Ok, rozumiem. Jeśli się nie konfliktuje to ok.